visualizations/frontend/other/TTable.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <div class="rootTableContainer">
     <slot name="columnConfig" :setColumnConfig="setColumnConfig"></slot>
```

Comment:
Let me know if this will cause an bugs to put the slot at the top of the template. The slot was being considered an element and although and empty div, it was creating a gap because it was being rendered in the DOM as `<div></div>. Moving it to the top of the template took it out of the `rootTableContainer` flex container to avoid the problem.

Reply:
I am flabbergasted that that works in vue 2, in vue 3 you can have multiple root elements but multiple root elements in vue 2 has always resulted in a compiler error. I wonder if something is different about slots? Just to be safe and assuming doing so doesn't mess up the styling, it would be a good idea to add a div around the whole template section so there is only one root element.